nmtwrapper_test.go

```diff
@@ -73,10 +75,10 @@ type constructor struct {
 	opts       []nmt.Option
 }
 
-// newConstructor creates a tree constructor function as required by rsmt2d to
+// newErasuredNamespacedMerkleTreeConstructor creates a tree constructor function as required by rsmt2d to
 // calculate the data root. It creates that tree using the
 // erasuredNamespacedMerkleTree.
-func newConstructor(squareSize uint64, opts ...nmt.Option) TreeConstructorFn {
+func newErasuredNamespacedMerkleTreeConstructor(squareSize uint64, opts ...nmt.Option) TreeConstructorFn {
```

Comment:
a bit unrelated to this PR, but do we have the wrapper in both places, here and in app? https://github.com/celestiaorg/celestia-app/tree/main/pkg/wrapper

Reply:
Good question! Yes, we have, I did it in the past to conduct some tests in the rsmt2d side without making circular dependency to the app. 
PS: There is also an open issue https://github.com/celestiaorg/rsmt2d/issues/248 about moving wrapper completely to the rsmt2d repo.